bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceScreen.kt

```diff
@@ -134,55 +145,88 @@ class DigitalInvoiceScreen {
         }
     }
 
-    fun verifyHelpTextOnNextScreen() {
+    fun verifyHelpTextOnNextScreen(): Boolean {
         val helpText = device.findObject(
             UiSelector()
                 .className("android.widget.TextView")
                 .text("Help")
         )
-        helpText.exists()
+        return helpText.exists()
     }
 
-    fun verifyFirstTitleOnHelpScreen() {
+    fun verifyFirstTitleOnHelpScreen(): Boolean {
+        var isFirstTitleDisplayed = false
         onView(
             allOf(withId(net.gini.android.bank.sdk.R.id.gbs_help_title),
                 withText("1. How does a digital invoice work?")
             )
         )
-            .check(matches(isDisplayed()))
+            .check { view, noViewFoundException ->
+                if (noViewFoundException == null || view.isShown()) {
+                    isFirstTitleDisplayed = true
+                }
+            }
+        return isFirstTitleDisplayed
     }
 
-    fun checkTotalTitleIsDisplayed() {
+    fun checkTotalTitleIsDisplayed(): Boolean {
+        var isTotalTitleDisplayed = false
         onView(
             allOf(withId(net.gini.android.bank.sdk.R.id.total_label),
                 withText("Total")
             )
         )
-            .check(matches(isDisplayed()))
+            .check { view, noViewFoundException ->
+                if (noViewFoundException == null || view.isShown()) {
+                    isTotalTitleDisplayed = true
+                }
+            }
+        return isTotalTitleDisplayed
     }
 
-    fun checkTotalPriceIsDisplayed() {
-        onView(withId(net.gini.android.bank.sdk.R.id.gross_price_total_integral_part))
-            .check(matches(isDisplayed()))
+    fun checkTotalPriceIsDisplayed() : Boolean{
+        var isTotalPriceDisplayed = false
+        onView((withId(net.gini.android.bank.sdk.R.id.gross_price_total_integral_part)))
+            .check { view, noViewFoundException ->
+                if (noViewFoundException == null || view.isShown()) {
+                    isTotalPriceDisplayed = true
+                }
+            }
+        return isTotalPriceDisplayed
     }
 
-    fun checkInitialPrice() {
+    fun storeInitialPrice() {
         onView(withId(net.gini.android.bank.sdk.R.id.gross_price_total_integral_part))
             .check { view, _ ->
                 val totalTextView = view as TextView
                 initialValue = totalTextView.text.toString()
             }
     }
 
-    fun checkUpdatedPrice() {
+    fun verifyTotalSumValue(): Boolean{
+        var isSumDifferent = false
+            if(initialValue!=updatedValue) {
+            isSumDifferent = true
+        }
+        return isSumDifferent
+//        assertNotEquals("Price should have changed after toggling", initialValue, updatedValue)
```

Comment:
[Request] Please remove commented code if not used anymore.

Reply:
done.